src/repository/models.py

```diff
@@ -401,6 +402,14 @@ class Preprint(models.Model):
     date_updated = models.DateTimeField(blank=True, null=True)
     current_step = models.IntegerField(default=1)
 
+    article = models.OneToOneField(
+        'submission.Article',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text='Linked article of this preprint.',
+    )
+
```

Comment:
We should probably also display this relationship in article archive and the preprint page.

Reply:
The relationship is already on the preprint page. From what I remeber there was no request for it to be on the article page though.